src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  width: calc($cell-size * 7 + $gap * 6);
```

Comment:
The variable `$cell-size` is used but not defined or imported in this file. Ensure that it is defined in a variables file or imported correctly.